pims/tests/test_display.py

```diff
@@ -155,6 +165,25 @@ def test_quality_h264(self):
 
         assert_less(compressed_size, lossless_size)
 
+    def test_rgba_h264(self):
+        """Remove alpha channel."""
+        # Start with smoke test for H.264
+        self.export_func(self.sequence_rgba, self.tempfile, codec='libx264')
+        # Check that RGB channels are preserved
+        self.export_func(self.sequence_rgba, self.tempfile, codec='rawvideo',
+                         pixel_format='bgr24')
+        sequence_rgba_stripped = self.sequence_rgba[:, :, :, :3]
+        with pims.open(self.tempfile) as reader:
+            assert_equal(len(reader), self.expected_len)
+            assert_equal(reader.frame_shape, self.expected_shape)
+            for a, b in zip(sequence_rgba_stripped, reader):
+                assert_array_equal(a, b)
+
+    def test_rgba_memorder(self):
+        """RGBA with alpha removed has different memory order."""
+        self.export_func(self.sequence_rgba[:, :, :, :3], self.tempfile,
```

Comment:
Maybe make this more explicit by imposing fortran order? Basically you can copy the `rawvideo export` test with the order change added, that will also test the result.

Reply:
Good idea. The revised test also catches the original bug.